dlp/snippets/deid.py

```diff
@@ -937,6 +937,76 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_simple_word_list]
+def deidentify_with_simple_word_list(
+    project: str,
+    input_str: str,
+    custom_info_type_name: str,
+    word_list: list[str],
```

Comment:
```suggestion
    word_list: List[str],
```

You'll also need to import List
`from typing import List`

I was wrong - `list[str]` can be used after 3.9 not 3.8 😅

Reply:
ohh okay, got it, thank you for your comment.

Reply:
updated please check